src/System Application/App/Guided Experience/src/Checklist Banner/ChecklistBanner.Codeunit.al

```diff
@@ -41,4 +41,15 @@ codeunit 3728 "Checklist Banner"
     internal procedure OnOpenChecklistBannerPage(var SkipWelcomeState: Boolean; IsEvaluationCompany: Boolean)
     begin
     end;
+
+    /// <summary>
+    /// Integration Event to provide the ability to filter on the checklist items before it is loaded to the checklist banner.
+    /// Do NOT use complex logic in the subscriber as it will impact the performance of opening company.
+    /// </summary>
+    /// <param name="SkipChecklistItem">Whether to skip the checklist item or not </param>
+    /// <param name="SpotlightTourType">Specifies the checklist's Spotlight Tour type if any</param>
+    [IntegrationEvent(false, false)]
+    internal procedure OnBeforeAddChecklistItemToBuffer(var SkipChecklistItem: Boolean; SpotlightTourType: Enum "Spotlight Tour Type")
```

Comment:
Missing the Checklist Item itself. Any subscriber would not know what they are skipping except for the type.

Reply:
Privacy deployment failed, so asked Sam directly.

The factbox is shown when there is read access to entity text, which is granted via `"System Application - View"`

So I guess it's safe to say in Eval Company, the Text Box we are looking for is always there. I vote we just remove the logic as the PR's current state